### PR TITLE
build(package): expose new swc_cli as binary

### PR DESCRIPTION
**Pull request by kwonoj**
**Description:**
Actually expose new cli implementation when install `@swc/core`. 

Note I choose binary name to `swcx`. It doesn't have strong meaning (ne`x`t? e`x`perimental?), only to avoid conflict with binary name installed via existing `@swc/cli`. Once this one is matured enough, we can plan migrate & depreacte existing `@swc/cli`.

**Related issue (if exists):**

- https://github.com/swc-project/swc/issues/1589

### CODE REVIEW

File: scripts/npm/android-arm64/package.json [hunk 1]
```diff
@@ -12,6 +12,9 @@
         "swc.android-arm64.node",
         "swc"
     ],
+    "bin": {
```
Comment (kwonoj) [inline scripts/npm/android-arm64/package.json:15]: this actually brings me a question: where do we use binary for `android`? and do we have to support /cli for this target?

Comment (kdy1) [reply scripts/npm/android-arm64/package.json:15]: Hmm... I'm not sure. Maybe termux?

cc @Brooooooklyn Do you have any idea?

Comment (Brooooooklyn) [reply scripts/npm/android-arm64/package.json:15]: Yes, seems like there are many termux users

Comment (kwonoj) [reply scripts/npm/android-arm64/package.json:15]: That.. is surprising. I'm curious if we can reduce some of matrix including this target. If we say many, do we know how many are there?

Comment (kwonoj) [reply scripts/npm/android-arm64/package.json:15]: (either cases, this PR won't deprecate anything yet)

**Review by kdy1 — APPROVED** [review]
LGTM. Thank you!

---

swc-bump:
 - ''

**Pull request merged**